apps/core/src/modules/user/signUp/routes/signUp.routes.ts

```diff
@@ -36,6 +36,7 @@ export default async function (app: FastifyInstance) {
     '/users/confirm',
     async (request: FastifyRequest<{ Body: { token: string } }>, reply) => {
       const { token } = request.body;
+      logger.info({ token }, 'token on confirm');
```

Comment:
why did you not use request.log as you used in other files?

Reply:
this log is dirty, and i`m gonna remove it in the next commit